Update example docker-compose.yaml network_mode: host

Dukat-Gul opened the pull request:
network_mode: host added to the vrmlogger section of the docker-compose.yaml example to read main interface MAC instead of virtual one - so that within vrm the physical MAC address of the eth0 is used each and everytime.

Dukat-Gul committed:
Update example docker-compose.yaml network_mode: host

network_mode: host added to the vrmlogger section of the docker-compose.yaml example to read main interface MAC instead of virtual one - so that within vrm the physical MAC address of the eth0 is used each and everytime.

Dukat-Gul committed:
Update entrypoint.sh - Add DC System and update frequency to VRM portal

Changes to the /data/conf/settings.xml which enable the DC system and change the default 900 second reporting to VRM portal to every 60 seconds.

Dukat-Gul committed:
Update Dockerfile - SSH added

Added openssh-client to apt command to reduce ssh rsa key errors in logging

iuriaranda commented on vrmlogger/entrypoint.sh:
```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 
 mkdir -p /data/conf /data/db
+sed -i 's+>900</LogInterval>+>60</LogInterval>+g' /data/conf/settings.xml
+sed -i 's+>0</HasDcSystem>+>1</HasDcSystem>+g' /data/conf/settings.xml
```
hmm I need to verify this, but afaik you can change these settings via MQTT (or dbus), which then get persisted in the `settings.xml` file. With this, these settings will always get overwritten when the container starts, which is not ideal if a user changes the defaults via MQTT.

iuriaranda reviewed (commented):
Thanks for the PR. Just a small comment on the default settings you set, which I need to test out and verify. Other than that LGTM

Dukat-Gul committed:
Update entrypoint.sh VRM remote MQTT

Preliminary addition of Mqtt to settings.conf to allow VRM access via remote MQTT - Work In Progress.
